src/Illuminate/Cache/PhpRedisLock.php

```diff
@@ -51,11 +51,11 @@ protected function serializedAndCompressedOwner(): string
         // https://github.com/phpredis/phpredis/issues/1938
         if ($this->compressed()) {
             if ($this->lzfCompressed()) {
-                $owner = \lzf_compress($owner);
+                $owner = lzf_compress($owner);
```

Comment:
Just wondering, why would it remove the \ from this line if the purpose of this fixer rule is to 'Add leading slash before optimized function invocation to speed up resolving.' according to their doc.

Not a reviewer or anything, just wonder :)

Reply:
because the config will only prefix `\` for global functions that can be improved via opcache.

Reply:
I see, thanks for the clarification!

Reply:
Considering that `lzf_compress` still has to be resolved via the namespace, is the result of that process cached in opcache (resulting in the same effect as prepending with \\), or is the performance effect of these calls just insignificant when compared to the functions that have special opcodes?